pandokia/import_data.py

```diff
@@ -279,7 +281,7 @@ def try_insert(self, db, key_id):
                  self.test_runner,
                  okf]
         return db.execute(
-            "INSERT INTO result_scalar ( %s test_run, host, project, test_name, context, status, start_time, end_time, location, attn, test_runner, has_okfile ) values "
+            "INSERT IGNORE INTO result_scalar ( %s test_run, host, project, test_name, context, status, start_time, end_time, location, attn, test_runner, has_okfile ) values "
```

Comment:
this could use a comment about the change made - what will be happening and be overwritten now

Reply:
Will do.